react/features/chat/components/AbstractChat.js

```diff
@@ -92,7 +92,9 @@ export function _mapDispatchToProps(dispatch: Dispatch<any>) {
          */
         _onSendMessage(text: string) {
             dispatch(sendMessage(text));
-        }
+        },
+
+        dispatch
```

Comment:
why is this necessary? When you connect() you shoudl already have dispatch(), otherwise you can just compose a new mapDispatchToProps